packages/shared/styles/components/organisms/SfHeader.scss

```diff
@@ -1,243 +1,174 @@
-      width: var(--header-navigation-item-width, 100%);
-      & > .sf-header-navigation-item__menu-item{
-        --menu-item-font-size: var(--font-size--base);
-        --menu-item-font-weight: var(--font-weight--medium);
-        --menu-item-text-transform: uppercase;
-        display: var(--header-navigation-item-menu-item-display, flex);
-        padding: var(--header-navigation-item-menu-item-padding, var(--spacer-sm));
-        margin: var(--header-navigation-item-menu-item-margin, 0);
-        @include border(--header-navigation-item-menu-item-border, 0 0 1px 0, solid, var(--c-light));
-      }
-    }
-  }
-  
-}
-.sf-header__navigation {
-  display: none;
-  &__menu {
-    display: var(--header-navigation-menu-display, none);
-    @include for-desktop {
-      --header-navigation-menu-display: flex;
-    }
-  }
-  &__sidebar {
-    --sidebar-content-padding: 0;
-    display: var(--header-navigation-sidebar-display, flex);
-    @include for-desktop {
-      --header-navigation-sidebar-display: none;
-    }
-  }
-}
-.sf-header {
-  position: relative;
-  &__wrapper {
-    position: var(--header-wrapper-position);
-    z-index: var(--header-wrapper-z-index, 1);
-    width: 100%;
-    background: var(--header-background, #ffffff);
-    transition: var(--header-wrapper-transition, transform 150ms ease-in-out);
-    transform: var(--header-wrapper-transform);
-    box-shadow: var(--header-box-shadow);
-  }
-  &__header {
-    display: flex;
-    flex-wrap: var(--header-flex-wrap, wrap);
-    align-items: center;
-    justify-content: center;
-    padding: var(--header-padding, 0 var(--spacer-sm));
-    max-width: var(--header-width, 77.5rem);
-    margin: var(--header-margin,0 auto);
-    @include for-desktop {
-      flex-wrap: var(--header-flex-wrap, nowrap);
+  &__logo {    
+    flex: 0 0 auto;
+    margin: var(--spacer-xs) 0;
+    z-index: 1;
+    .sf-image {
+      width: auto;
+      height: 2.125rem;
     }
   }
-  &__logo {
-    --image-width: var(--header-logo-width, auto);
-    --image-height: var(--header-logo-height, 2.125rem);
-    flex: 0 0 var(--header-logo-width, auto);
-    margin: var(--header-logo-margin, var(--spacer-xs) 0);
-  }
   &__title {
-    margin: var(--header-title-margin, var(--spacer-xs) 0);
+    margin: var(--spacer-xs) 0;
   }
   &__aside:not(:empty) {
-    margin: var(--header-aside-margin, 0 0 0 auto);
+    margin: 0 0 0 auto;
   }
   &__actions {
     flex: 100%;
     display: flex;
-    flex-wrap: var(--header-actions-flex-wrap, wrap);
+    flex-wrap: wrap;
     align-items: center;
     justify-content: flex-end;
     @include for-desktop {
-      flex-wrap: var(--header-actions-flex-wrap, nowrap);
+      flex-wrap: nowrap;
     }
   }
   &__navigation {
+    --sidebar-top: auto;
+    --sidebar-bottom: auto;
+    display: flex;
     order: 1;    
-    flex: 0 0 calc(100% + var(--spacer-sm) * 2);
-    margin: var(--header-navigation-margin, 0 calc(var(--spacer-sm) * -1));
```

Comment:
suggestion: we should also add some info to migration guide about variables that are no longer valid

Reply:
Yes, they are added.